packages/spor-react/src/datepicker/DateTimeSegment.tsx

```diff
@@ -38,7 +38,6 @@ export const DateTimeSegment = forwardRef<HTMLDivElement, DateTimeSegmentProps>(
           ...segmentProps.style,
           fontVariantNumeric: "tabular-nums",
           boxSizing: "content-box",
-          color: colors.darkGrey,
```

Comment:
This unfortunately overwrote the focused style as well, so moved it to `dateTimeSegment:` instead